gateway-service/src/test/java/org/zowe/apiml/acceptance/SafIdtSchemeTest.java

```diff
@@ -64,7 +92,7 @@ void thenValidTokenIsProvided() throws IOException {
                 ArgumentCaptor<HttpUriRequest> captor = ArgumentCaptor.forClass(HttpUriRequest.class);
                 verify(mockClient, times(1)).execute(captor.capture());
 
-                assertHeaderWithValue(captor.getValue(), "X-SAF-Token", "validToken" + validJwtToken.getValue());
+                assertHeaderWithValue(captor.getValue(), "X-SAF-Token", resultSafToken);
```

Comment:
Maybe it would be better if we reuse Authorization header and bearer token type, that it would better express our intent.

Reply:
We can have a discussion but we had the same discussion when this was actually introduced.